### PR TITLE
LaTeX support thanks to MathJax

### PR DESCRIPTION
**Pull request by Technici4n**
I'm satisfied with the result. What do you think about it @RobinJadoul ? It might be a bit slow, but I'm afraid we can't do much for it.

### CODE REVIEW

**Comment by RobinJadoul** [comment]
`$1 \lt 2~\&\&~2 < 3$` works in the preview, but shows an error on the actual rendering.
Maybe some problem with escaping?

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,14 +14,13 @@
...
-				extensions: ["tex2jax.js", "fast-preview.js"],
+				extensions: ["tex2jax.js", "Safe.js", "fast-preview.js"],
 				tex2jax: {
 					inlineMath: [['$','$'], ['\\(','\\)']],
 					processEscapes: true // Allow \$ for a literal $
 				},
-				jax: ["input/TeX", "output/HTML-CSS"],
-				messageStyle: "none",
-				showMathMenu: false,
+				showMathManu: true,
```
Comment (RobinJadoul) [inline app/views/layouts/application.html.erb]: I added those settings to keep it as non-intrusive as possible.
If you want them, I'm fine with that, but then you might want to change `showMathManu` to `showMathMenu`

**Comment by RobinJadoul** [comment]
Apart from these 2 comments, it feels okay for me

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Technici4n) [reply app/views/layouts/application.html.erb]: Actually, we can have a look at the code to see math so it's not necessary to show the menu.

Comment (Technici4n) [reply app/views/layouts/application.html.erb]: For now, it can be removed indeed.

**Comment by RobinJadoul** [comment]
I can't edit or merge, so I suggest you change the settings (or not) and merge this one, since it seems to work okay?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -14,23 +14,25 @@
...
+				extensions: ["tex2jax.js", "Safe.js"],
 				tex2jax: {
 					inlineMath: [['$','$'], ['\\(','\\)']],
 					processEscapes: true // Allow \$ for a literal $
 				},
-				showMathManu: true,
-				jax: ["input/TeX", "output/HTML-CSS"]
+				jax: ["input/TeX", "output/HTML-CSS"],
+				messageStyle: "none",
+				showMathManu: false,
```
Comment (RobinJadoul) [inline app/views/layouts/application.html.erb]: `showMathMenu`

Comment (Technici4n) [reply app/views/layouts/application.html.erb]: What ?

**Comment by RobinJadoul** [comment]
It should be menu instead if manu

**Comment by Technici4n** [comment]
Indeed. Now there is an issue with mobile phones. JS is not executed. Have a look at Gitter

**Pull request merged**